src/trix/controllers/toolbar_controller.coffee

```diff
@@ -84,6 +84,11 @@ class Trix.ToolbarController extends Trix.BasicObject
       else
         element.classList.remove("active")
 
+      if not @delegate.composition.canAddBlockAttribute(attributeName)
+        element.disabled = true
+      else
+        element.disabled = false
```

Comment:
Rather than reach out to ask what attributes can be added, let's find a way to pass that information to the toolbar controller [like we do for current attributes and actions](https://github.com/basecamp/trix/blob/master/src/trix/controllers/editor_controller.coffee#L51-L53).